query-creator/create-query/steps/aws.js

```diff
@@ -301,20 +304,35 @@ learners_and_answers AS ( SELECT run_remote_endpoint remote_endpoint, runnable_u
       });
       allColumns.push(...questionsColumns);
 
+      let columnWithAnswerText = allColumns.find(column => column.name.includes("_text"));
+      orderByText += columnWithAnswerText ? `\nORDER BY learner_id NULLS FIRST, ${columnWithAnswerText.name}`: "";
+
```

Comment:
I think this will have a problem when Doug's multiple activities report is included.
In that case there will need to be multiple learner_id fields for each "student+class" row. 
If a student hasn't done one of the activities then that learner_id field will probably be null.

Perhaps you can use the `class` column instead. Those should always be non-null.

If there are no open response questions in the activity then ordering seems like it will be messed up. You could populate the first column (currently remote_endpoint) with a fixed string like "correct answer" for this row. And then perhaps put "prompt" in that column for the prompts row. Now could could use that column as the secondary ordering instead of `columnWithAnswerText`. You'll have to use `DESC` since `correct answer` is before `prompt`.